corona-warn-app/cwa-app-android
# Vaccination legal text consent screen (EXPOSUREAPP-6725)

- Legal text consent screen (texts are not final)
### Testing
- Click on "Impfzertifikat registrieren" to see the consent screen
- Use Test Menu > Onboarding Options > Vaccination Registration Onboarding and restart the app to see it again
